consensus/mint-client/src/bin/main.rs

```diff
@@ -17,16 +17,34 @@ use mc_transaction_core::{
     constants::MAX_TOMBSTONE_BLOCKS,
     mint::{MintConfigTx, MintTx},
 };
-use mc_util_grpc::ConnectionUriGrpcioChannel;
+use mc_util_grpc::{ConnectionUriGrpcioChannel, CHAIN_ID_GRPC_HEADER};
 use protobuf::ProtobufEnum;
 use std::{fs, process::exit, sync::Arc};
 
+// Make a "call option" object which includes appropriate grpc headers
+fn call_option(chain_id: &str) -> CallOption {
```

Comment:
Seems like this is a thing that will be repeated in many places, maybe it should go into `mc_util_grpc`?
(in a separate PR on master)

Reply:
I created a ticket for this, thanks